freqtrade/rpc/fiat_convert.py

```diff
@@ -25,6 +27,7 @@ class CryptoToFiatConverter:
     _coingekko: CoinGeckoAPI = None
 
     _cryptomap: Dict = {}
+    _backoff = int
```

Comment:
The only open point is whether or not the `_backoff` attribute here is okay.
Maybe this should be 
```suggestion
    _backoff: int = 0
```
instead?

Reply:
the way it is now is definitely wrong (it's assigning the type of int to the variable ...) i wonder why mypy ain't complaining very loudly about this.

When doing that however, i think you must also check for `if self._backoff and ...` below - otherwise `self._backoff` will always be < current timestamp.